mava/types.py

```diff
@@ -118,6 +129,13 @@ class LearnerState(NamedTuple):
     timestep: TimeStep
 
 
+class SebulbaLearnerState(NamedTuple):
```

Comment:
Same here, can we not use the existing `LearnerState` it will make the mava API much cleaner because then everything will return a `LearnerState`/`RnnLearnerState`